guides/source/caching_with_rails.md

```diff
@@ -295,14 +295,21 @@ Consider the following example. An application has a `Product` model with an ins
 ```ruby
 class Product < ApplicationRecord
   def competing_price
-    Rails.cache.fetch("#{cache_key}/competing_price", expires_in: 12.hours) do
+    key = cache_key              # => products/233
+
+    key = cache_key_with_version # => products/233-20140225082222765838000
```

Comment:
This code is artificial to showcase the methods. This is not how we write the guides. I think we should only use `cache_key_with_version` here.

Reply:
@rafaelfranca Thanks! Fix the code to use only `cache_key_with_version` method and also fixed the note.